apps/remix-ide/src/app/panels/file-panel.js

```diff
@@ -1,18 +1,18 @@
-import { ViewPlugin } from '@remixproject/engine-web'
-
-import * as packageJson from '../../../../../package.json'
-import React from 'react' // eslint-disable-line
-import ReactDOM from 'react-dom'
-import { Workspace } from '@remix-ui/workspace' // eslint-disable-line
-import * as ethutil from 'ethereumjs-util'
-import { checkSpecialChars, checkSlash } from '../../lib/helper'
-var EventManager = require('../../lib/events')
-var { RemixdHandle } = require('../files/remixd-handle.js')
-var { GitHandle } = require('../files/git-handle.js')
-var globalRegistry = require('../../global/registry')
-var examples = require('../editor/examples')
-var GistHandler = require('../../lib/gist-handler')
-var QueryParams = require('../../lib/query-params')
+import { ViewPlugin } from '@remixproject/engine-web';
+
+import * as packageJson from '../../../../../package.json';
+import React from 'react'; // eslint-disable-line
+import ReactDOM from 'react-dom';
+import { Workspace } from '@remix-ui/workspace'; // eslint-disable-line
+import * as ethutil from 'ethereumjs-util';
+import { checkSpecialChars, checkSlash } from '../../lib/helper';
+var EventManager = require('../../lib/events');
+var { RemixdHandle } = require('../files/remixd-handle.js');
+var { GitHandle } = require('../files/git-handle.js');
+var globalRegistry = require('../../global/registry');
+var examples = require('../editor/examples');
+var GistHandler = require('../../lib/gist-handler');
+var QueryParams = require('../../lib/query-params');
```

Comment:
no need for ';'
some linters are not allowing them

Reply:
removed semi colons too